test/test_ff_utils.py

```diff
@@ -285,7 +285,7 @@ def test_stuff_in_queues(integrated_ff):
     # just take the first handful
     for item in search_res[:8]:
         ff_utils.patch_metadata({}, obj_id=item['uuid'], key=integrated_ff['ff_key'])
-    time.sleep(5)  # let queues catch up
+    time.sleep(3)  # let queues catch up
```

Comment:
I wish we did something more abstract like `let_queues_catch_up()` even if it was just implemented as a sleep, though I suppose we do different amounts of sleep but I suppose that's a problem for another day. What you're doing here is pretty common.